ieeeuoft/hackathon-template
# IEEE-177 Maintain filters when drawer opens and closes

## Overview

- Resolves #[IEEE177](https://ieeeuoft.atlassian.net/jira/software/projects/IEEE/boards/1?selectedIssue=IEEE-177)
- Filters are maintained even when drawer opens and closes. This happens when the screen size is reduced to mobile ratio


## Unit Tests Created

- None


## Steps to QA

- Navigate to the Inventory Page
- Reduce the screen size to moile ratio
- Open the Filter Drawer
- Select Filters and then close the Drawer
- Reopen the Filter Drawer and the Filters should be maintained
